### ISSUE TITLE
Changing 'disable' property binding from true to false prevents tags being added to selection

### ISSUE THREAD

**Issue by needsmorecoffee** [Gbuomprisco/ngx-chips#620]
PLEASE MAKE SURE THAT:
- you searched similar issues online (9/10 issues in this repo are solved by googling, so please...do it)
- you provide an online demo I can see without having to replicate your environment
- you help me by debugging your issue, and if you can't, do go on filling out this form

**I'm submitting a ...**  (check one with "x")
```
[X ] bug report => search github for a similar issue or PR before submitting
[ ] support request/question

Notice: feature requests will be ignored, submit a PR if you'd like
```

**Current behavior**

Bind [disable] to a variable, set to true initially and chips input is disabled as expected. When bound variable is set to false, the input box regains interactivity correctly, but when a chip is submitted, it does not get added to the list of chips.

**Expected behavior**

When disable flag is set back to false, the full functionality should be retained.

**Minimal reproduction of the problem with instructions (if applicable)**

As above.

**What do you use to build your app?. Please specify the version**

angular-cli 1.5.0

**Angular version:** 

4.4.6

**ngx-chips version:**

1.5.10

**Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

all

**Comment by pacoma93** [comment]
Have the same dilemma, but i did find some kind of solution to it. You can add a tag(div perhaps?) around the input-tag and autocomplete, then use css: pointer-event: none; to disable the functions. using [ngClass] to toggle the disable.

**Comment by field123** [comment]
+1

**Comment by kotmatpockuh** [comment]
+1

**Comment by Gbuomprisco** [comment]
this should be fixed with version 1.6.8
